src/containers/Account/ProfilePage.js

```diff
@@ -24,8 +24,11 @@ function ProfilePage(props) {
     updateProfile,
     userLmsSettings,
     getUserLmsSettings,
+    loadOrganizationTypes,
   } = props;
 
+  const organizationTypes = useSelector((state) => state.auth.organizationTypes);
```

Comment:
Do we have organizationTypes into auth reducer?

Reply:
yes,  it is the part of the registration